src/components/ResumeBuilder/Confirmation.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { useRouter } from 'next/router'
+import CheckCircle from '../../images/check-circle.png'
+import '../../assets/css/resumebuilder.css'
```

Comment:
[This](https://github.com/Vets-Who-Code/vets-who-code-app/pull/421/files#diff-685933f525b831374b30df1f8a8bd36f88ba49e5b34bcbb64203eaaceedea5c1R16) is global. I think these import statements could be dropped.